srv/SpawnEntity.srv

```diff
@@ -0,0 +1,32 @@
+# Spawn an entity (a robot, other object) by name or URI
+
+string name                             # A name to give to the spawned entity.
+                                        # If empty, a name field in the uri file or resource_string will be used,
+                                        # if supported and not empty (e.g. "name" field in SDFormat, URDF).
+                                        # If the name is still empty or not unique (as determined by the simulator),
+                                        # the service returns a generated name in the entity_name response field if the
+                                        # allow_renaming field is set to true. Otherwise, the service call fails and an
+                                        # error is returned in the status_message.
+bool allow_renaming                     # Determines whether the spawning succeeds with a non-unique name.
+                                        # If it is set to true, the user should always check entity_name response field
+                                        # and use it for any further interactions.
+string uri                              # Resource such as SDFormat, URDF, USD or MJCF file, a native prefab, etc.
+                                        # Valid URIs can be determined by calling GetSpawnables first, and you can check
+                                        # the simulator format support by reading SimulatorFeatures spawn_formats field.
+                                        # If uri field is empty, resource_string must not be empty.
+string resource_string                  # An entity definition file passed as a string.
+                                        # Simulators may support spawning from a file generated on the fly (e.g. XACRO).
+                                        # Check whether it is supported by your simulator through GetSimulatorFeatures.
```

Comment:
Consider stating that if both `uri` and `resource_string` are given, `uri` will be used.